src/parse/mod.rs

```diff
@@ -87,7 +92,8 @@ fn is_key_char(
   ch: &str,
 ) -> bool {
   match ch {
-    " " | "!" | ")" | "\"" | "'" | "]" => true,
+    " " | "\n" | "\t" | "!" | ")" | "]" | "}" | "\"" | "'" | "#" => true,
```

Comment:
We should probably split these into separate line by category for readability soon, but not required for this PR.

(I.e. whitespace (including `,`) vs literal terminators vs special (`!` and `#`))

Reply:
Should probobly be split into a few different functions tbh. Cause now it's doing a few different things

Checking for characters which should be ignored, but seperate "things"
Defines all chars which can't be used in labels. ( This means that h&*$^=:;({[<>~\b is a valid label (or well, unsure of the backspace))
And it defines special chars used for other things which it would be ugly to require an whitespace before

Reply:
Very true.